perf/benchmark/runner/runner.py

```diff
@@ -117,42 +118,36 @@ def __init__(
         else:
             sys.exit("invalid mesh %s, must be istio or linkerd" % mesh)
 
-    def nosidecar(self, fortio_cmd):
-        basestr = "http://{svc}:{port}/echo?size={size}"
-        if self.mode == "grpc":
-            basestr = "-payload-size {size} {svc}:{port}"
-        return fortio_cmd + "_base " + basestr.format(
-            svc=self.server.ip, port=self.ports[self.mode]["direct_port"], size=self.size)
+    def get_protocol_uri_fragment(self):
+        return "https" if self.mode == "grpc" else "http" 
 
-    def serversidecar(self, fortio_cmd):
-        basestr = "http://{svc}:{port}/echo?size={size}"
-        if self.mode == "grpc":
-            basestr = "-payload-size {size} {svc}:{port}"
-        return fortio_cmd + "_serveronly " + basestr.format(
-            svc=self.server.ip, port=self.ports[self.mode]["port"], size=self.size)
+    def nosidecar(self):
+        basestr = "{protocol}://{svc}:{port}/"
+        return "base", basestr.format(
+            svc=self.server.ip, port=self.ports[self.mode]["direct_port"], protocol=self.get_protocol_uri_fragment())
 
-    def clientsidecar(self, fortio_cmd):
-        basestr = "http://{svc}:{port}/echo?size={size}"
-        if self.mode == "grpc":
-            basestr = "-payload-size {size} {svc}:{port}"
-        return fortio_cmd + "_clientonly " + basestr.format(
-            svc=self.server.labels["app"], port=self.ports[self.mode]["direct_port"], size=self.size)
+    def serversidecar(self):
+        basestr = "{protocol}://{svc}:{port}/"
+        return "serveronly", basestr.format(
+            svc=self.server.ip, port=self.ports[self.mode]["port"], protocol=self.get_protocol_uri_fragment())
 
-    def bothsidecar(self, fortio_cmd):
-        basestr = "http://{svc}:{port}/echo?size={size}"
-        if self.mode == "grpc":
-            basestr = "-payload-size {size} {svc}:{port}"
-        return fortio_cmd + "_both " + basestr.format(
-            svc=self.server.labels["app"], port=self.ports[self.mode]["port"], size=self.size)
+    def clientsidecar(self, nighthawk_cmd):
+        basestr = "{protocol}://{svc}:{port}/"
+        return "base", basestr.format(
+            svc=self.server.ip, port=self.ports[self.mode]["direct_port"], protocol=self.get_protocol_uri_fragment())
```

Comment:
`svc` either `server.ip` or service name should be fine

Reply:
To make sure I understand correctly, this is ok as-is right?

Reply:
yes, it is ok, but prefer to be consistent with the old one, which is using service name

Reply:
Done in 7a372c24f9bbcb84a97daf4b1b444a32cb710947